src/transactions/utils/getTransactionBytes.js

```diff
@@ -179,7 +180,16 @@ const getTransactionBytes = transaction => {
 			)
 		: Buffer.alloc(BYTESIZES.RECIPIENT_ID);
 
-	const transactionAmount = bignum(amount).toBuffer({
+	const amountBigNum = bignum(amount);
+	if (amountBigNum.lt(0)) {
```

Comment:
is `0` allowed considering the fee?

Reply:
Amount !== fee - but in general that is a good question.

Only type 0, 6 and 7 transactions have a configurable amount, else it will be default to 0.